kyo-prelude/shared/src/main/scala/kyo/kernel/Boundary.scala

```diff
@@ -22,19 +22,6 @@ import scala.quoted.*
   * @tparam S
   *   The source effects that must be handled before crossing
   */
-@implicitNotFound("""
-Could not create an Async boundary for effects: ${Ctx}
```

Comment:
This scenario doesn't happen anymore after https://github.com/getkyo/kyo/pull/850